azext_iot/tests/test_iot_ext_int.py

```diff
@@ -498,6 +498,16 @@ def test_hub_device_twins(self):
                          self.check('properties.desired.temperature.max', 100),
                          self.check('tags.location.region', 'US')])
 
+        self.cmd('iot hub distributed-tracing show -d {} -n {} -g {}'.format(device_ids[2], LIVE_HUB, LIVE_RG),
+                 checks=self.is_empty())
+
+        result = self.cmd('iot hub distributed-tracing update -d {} -n {} -g {} --sm on --sr 50'.format(device_ids[2],
+                          LIVE_HUB, LIVE_RG)).get_output_in_json()
+        assert result['Device ID'] == device_ids[2]
+        assert result['Sampling Mode'] == 'enabled'
+        assert result['Sampling Rate'] == '50%'
+        assert result['Sync Status'] == 'not in sync'
```

Comment:
Shouldn't there be a test for the 'in sync' state as well?

Reply:
"in sync" status can be possible when twin[reported] having the same values as twin[desired], Can we change the twin[reported] values? So that we can add a new test too.

Reply:
You can use the paho MQTT provider to do that. But we can have it in a subsequent commit.